### PR TITLE
A.1.22 A.2.22 Delete Profile picture

### PR DESCRIPTION
**Pull request by y1bs**
# Feature
A.1.22/ A.2.22 Delete Profile picture

# Changelog:

# Merge/Review Order

These other PRs should be merged or reviewed first

This PR blocks these other PRs

## Checklist:

- [ ] Merged latest develop
- [ ] PR title makes sense

### CODE REVIEW

File: src/redux/user/sagas.js [hunk 1]
```diff
@@ -432,6 +433,42 @@ export function* DELETE_ACCOUNT({ payload }) {
...
+    if (updatedUser) {
+      yield putResolve({
+        type: 'user/SET_STATE',
+        payload: {
+          ...updatedUser,
+        },
+      })
+      jwt.updateLocalUserData(updatedUser)
+
+      notification.success({
```
Comment (rawrrencee) [inline src/redux/user/sagas.js]: best if can call showNotification util  method, if not nvm too!

**Review by rawrrencee — DISMISSED** [review]
approved with one recommended minor change for notification function

File: src/redux/user/sagas.js (hunk 1 above)
Comment (y1bs) [reply src/redux/user/sagas.js]: Updated le

**Review by rawrrencee — APPROVED** [review]
thanks!!!

**Pull request merged**